lib/chef/resource/aws_cache_cluster.rb

```diff
@@ -0,0 +1,31 @@
+require 'chef/provisioning/aws_driver/aws_resource'
+
+class Chef::Resource::AwsCacheCluster < Chef::Provisioning::AWSDriver::AWSResource
+  aws_sdk_type AWS::ElastiCache
```

Comment:
I think its worth a comment explaining that there isn't actually a resource class